Makefile

```diff
@@ -20,6 +20,7 @@ PROMTOOL=$(abspath $(TOOLS_BIN_DIR)/promtool)
 
 GO_GCFLAGS ?= -gcflags=all='-N -l'
 GO=GO111MODULE=on GOFLAGS=-mod=vendor go
+GO_NO_VENDOR=GO111MODULE=on GOFLAGS=-mod=mod go
```

Comment:
This doesn't look like it's used anywhere

Reply:
ohh that was my bad. let me remove it